rules/restrict-access-to-the-control-plane-endpoint/rule.metadata.json

```
@@ -0,0 +1,25 @@
+{
+    "name": "restrict-access-to-the-control-plane-endpoint",
+    "attributes": {
+      "armoBuiltin": true,
+      "hostSensorRule": "false",
+      "imageScanRelated": false
+    },
+    "ruleLanguage": "Rego",
+    "match": [
```

```suggestion
    "dynamicMatch": [
```
`match` is used for k8s resources, when we need resources from a different source (cloud env, host-scanner) we use `dynamicMatch`